corehq/apps/users/templates/users/mobile_workers.html

```diff
@@ -0,0 +1,474 @@
+{% extends "style/bootstrap3/base_section.html" %}
+{% load i18n %}
+{% load hq_shared_tags %}
+{% load crispy_forms_tags %}
+{% load djangular_tags %}
+
+{% block js %}
+<script type="text/javascript"
+        src="{% new_static 'style/js/pagination.ng.js' %}"></script>
+<script type="text/javascript"
+        src="{% new_static 'users/js/mobile_workers.ng.js' %}"></script>
+{% endblock %}
+
+{% block js-inline %}{{ block.super }}
+<script type="text/javascript">
+$(function () {
+   'use strict';
+    $('#newMobileWorkerModal').on('shown.bs.modal', function () {
+        $('#id_username').focus();
```

Comment:
I don't want it to focus when it's on page load, but specifically when the modal is shown. Autofocus is for page load